spec/book_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe Book do
+  before :each do
+    @book = Book.new('Title', 'Author')
+    @person = Person.new(22, 'Name', parent_permission: true)
+  end
+
+  describe '#new' do
+    it 'takes two parameters and returns a Book object' do
+      expect(@book).to be_an_instance_of(Book)
+    end
+  end
+
+  describe '#as_json' do
+    test_data = {
+      title: 'Title',
+      author: 'Author'
+    }
+    it 'take a book instance an return it as json' do
+      expect(@book.as_json(@book)).to eq(test_data)
+    end
+  end
+
```

Comment:
- [ ] It would be better if we test our `to_json` method since we are suppose to test all the methods in our classes